src/routes/docs/components/forms.md

```diff
@@ -83,29 +83,34 @@ With the Input component, you can add <a href="https://flowbite-svelte.vercel.ap
 
 ```svelte example
 <script>
-  import { Label, Input } from 'flowbite-svelte'
+  import { Label, Input, InputAddon, ButtonGroup } from 'flowbite-svelte';
+  import { EnvelopeSolid } from 'flowbite-svelte-icons';
 </script>
 
 <div class="mb-6">
   <Label for="input-group-1" class="block mb-2">Your Email</Label>
-  <Input id="email" type="email" placeholder="name@flowbite.com">
-  <svg slot="left" aria-hidden="true" class="w-5 h-5 text-gray-500 dark:text-gray-400" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path d="M2.003 5.884L10 9.882l7.997-3.998A2 2 0 0016 4H4a2 2 0 00-1.997 1.884z"></path><path d="M18 8.118l-8 4-8-4V14a2 2 0 002 2h12a2 2 0 002-2V8.118z"></path></svg>
-  </Input>
+  <ButtonGroup class="w-full">
+    <InputAddon id="email" type="email" placeholder="name@flowbite.com">
```

Comment:
This is a different example. It's not the `InputAddon` here but the floating icon. So `slot="left"` should be used at `Envelope`.

Reply:
Thanks. Updated.